docs/source/nuget/packages.rst

```diff
@@ -264,6 +264,69 @@ And like this for Http API::
         }
     }
 
+Voxel.MiddyNet.HttpJsonBodyParser
+---------------------------------
+This package contains a middleware that parse JSON object mapping to a explicit type. There are two versions avalaible: 
```

Comment:
This package contains a middleware that parses a JSON to an object of an explicit type.

Reply:
Thanks!!! 🙏